client/src/models/solid/import-step-2.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiNoHistory, solid } from '@buerli.io/headless'
+import data from '../../resources/solid/AWVLogoCube.stp'
+import { Create, Param } from '../../store'
+
+export const paramsMap: Param[] = [].sort((a, b) => a.index - b.index)
+
```

Comment:
I think `sort` isn't needed here...

Reply:
Of course. This code is in the templates to see where params have to be defined and there is always this sort function appended. In this case it's actually not necessary